src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/quotes */ import React from "react";
 
-import './MovieCard.scss';
+import "./MovieCard.scss";
+import { Movie } from "../../types/Movie";
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+interface Props {
```

Comment:
almost there, just change "Props" to something more descriptive, e.g. MovieCardProps